contracts/cw-core/Cargo.toml

```diff
@@ -45,8 +45,7 @@ cw-paginate = { version = "0.2.0", path = "../../packages/cw-paginate" }
 
 [dev-dependencies]
 cosmwasm-schema = { version = "1.0.0" }
-# TODO: We are using the main branch of cw-multi-test until the next release, because we need recent stargate parsing fixes
-cw-multi-test = { git = "https://github.com/CosmWasm/cw-plus", branch = "main",  features = ["stargate"] }
+cw-multi-test = { version = "0.14", features = ["stargate"] }
```

Comment:
Am surprised this doesn't break with us using `0.13` for other cw-plus packages. If it works though I'm happy. :)

We'll unify all our dependencies before we release v2.

Reply:
Agreed, I just wanted to get the build / cargo update working again and we can use 0.14 everywhere in another PR.